File: apcs/e286/e286.cpp
#include <iostream>
using namespace std;
int main(){
	int MAP[4][4];
	int sum[4];
	int n=4;
	int sumi=0; 
	for (int i=0;i<n;i++){
		for(int j=0;j<n;j++)
		{
			cin >> MAP[i][j];
			sumi+=MAP[i][j];
		}
		sum[i]=sumi;
		sumi=0;
	}
	cout << sum[0] << ":" << sum[1] << endl;
	cout << sum[2] << ":" << sum[3] << endl;
	if ((sum[0]>sum[1]) and (sum[2]>sum[3]))	cout << "Win";
	else if ((sum[0]<sum[1]) and (sum[2]<sum[3]))	cout <<"Lose";
	else	cout << "Tie";
	return 0;
}

File: apcs/e286/e286.py
# 讀入測資字串，切割後，轉整數，再轉為list
home1    = sum( list( map(int, input().split() ) ) ) # 主隊第1場得分 
guest1   = sum( list( map(int, input().split() ) ) ) # 客隊第1場得分
home2    = sum( list( map(int, input().split() ) ) )  # 主隊第2場得分
guest2   = sum( list( map(int, input().split() ) ) )  # 客隊第2場得分
home_win = 0  # 主隊贏幾場
ans = ["Lose" , "Tie" , "Win"]
print( f"{home1}:{guest1}" )  # 輸出第1場兩隊比分
print( f"{home2}:{guest2}" )  # 輸出第2場兩隊比分

if home1 > guest1 :  # 如果主隊贏第一場
    home_win+=1   # 主隊贏場數+1

if home2 >guest2 :  # 如果主隊贏第二場
    home_win+=1   # 主隊贏場數+1
print(ans[home_win])

